middleman_protocol/middleman_protocol/concent_golem_messages/message.py

```diff
@@ -21,9 +23,16 @@ class NonceAbstractMessage(Message):
         'nonce',
     ] + Message.__slots__
 
+    def slots(self) -> List[List[Union[str, bytes, int]]]:
+        slots = super(NonceAbstractMessage, self).slots()
+        if ['nonce', self.nonce] in slots:  # pylint: disable=no-member
+            slots[slots.index(['nonce', self.nonce])][1] = str(self.nonce)  # pylint: disable=no-member
```

Comment:
Is this really necessary? Maybe it is OK to keep internally `nonce` as int, and add here property too for getting is as string? But if there is a case that it is required for this conversion to happen in `slots()` please let me know.

Reply:
In one of few last updates golem_messages Golem Team changed `cbor2` library to `cbor`, because it's faster than `cbor2`. This library has issues with serialization big integers and that's why we need to convert it during serialization and opposite thing during deserialization. We in our code don't need `nonce`, `v`, `r` or `s` as a string, but this serialization needs it.